extensions/sql-database-projects/src/common/uiConstants.ts

```diff
@@ -20,5 +20,6 @@ export namespace cssStyles {
 	export namespace fontStyle {
 		export const normal = 'normal';
 		export const italics = 'italic';
+		export const bold = 'bold';
```

Comment:
I see this added, but not seeing it added anywhere.  Should something be bolded now?

Reply:
This is a leftover of what I was trying for making the labels/titles bolded- but that didn't work. I will remove it in next commit.